convert/src/vertex_to_bd/models/vertex_models.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel
+
+from .common import S3Path
+
+
+class VertexRequest(BaseModel):
+    input_annotation: S3Path
+    output_pdf: Optional[S3Path]
+    output_tokens: Optional[S3Path]
+    output_annotation: Optional[S3Path]
+
+
+class BadgerdocToVertexRequest(BaseModel):
+    input_tokens: S3Path
+    input_annotation: S3Path
+    output_annotation: S3Path
+
+
+class Value(BaseModel):
+    start: int
+    end: int
+    text: str
+    labels: List[str]
+
+
+class ResultItem(BaseModel):
+    value: Optional[Value] = None
+    id: str = None
+    from_name: Optional[str] = None
+    to_name: Optional[str] = None
+    type: str
+    origin: Optional[str] = None
+    from_id: Optional[str] = None
+    to_id: Optional[str] = None
+    direction: Optional[str] = None
+    labels: Optional[List] = None
+
+
+class Value1(BaseModel):
+    start: int
+    end: int
+    score: float
+    text: str
+    labels: List[str]
+
+
+class ResultItem1(BaseModel):
+    id: str
+    from_name: str
+    to_name: str
+    type: str
+    value: Value1
+
+
+class Prediction(BaseModel):
+    id: int
+    model_version: str
+    created_ago: str
+    result: List[ResultItem1]
+    score: float
+    cluster: Any
+    neighbors: Any
+    mislabeling: float
+    created_at: str
+    updated_at: str
+    task: int
+
+
+class Annotation(BaseModel):
+    id: int = 7
+    completed_by: int = 1
+    result: List[ResultItem]
+    was_cancelled: bool = False
+    ground_truth: bool = False
+    created_at: str = "2022-12-13T09:57:08.451845Z"
+    updated_at: str = "2022-12-13T09:57:08.451875Z"
```

Comment:
I don't think these default values shoud be hardcoded.
If it is used to see value example in swagger - you could use Field(..., example="2022-12-13T09:57:08.451845Z") instead

Reply:
It shouldn't be hardcoded
Annotation part is mostly POC

Reply:
https://github.com/epam/badgerdoc/issues/170